### PR TITLE
feat(ingest): switch telemetry endpoint to Mixpanel

### PR DESCRIPTION
**Pull request by kevinhu**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by shirshanka — APPROVED** [review]
LGTM

File: metadata-ingestion/src/datahub/telemetry/telemetry.py [hunk 1]
```diff
@@ -42,6 +42,16 @@ def __init__(self):
         else:
             self.load_config()
 
+        # send updated user-level prop{erties
+        mp.people_set(
```
Comment (cccs-eric) [inline metadata-ingestion/src/datahub/telemetry/telemetry.py:46]: Seems like this line is by-passing the `self.enabled` flag.  The code should check that flag before calling mixpanel.  I have updated my fork to `HEAD` and ran into a proxy issue (my org is blocking mixpanel).

Comment (cccs-eric) [reply metadata-ingestion/src/datahub/telemetry/telemetry.py:46]: @kevinhu @shirshanka See my comment above.

Comment (kevinhu) [reply metadata-ingestion/src/datahub/telemetry/telemetry.py:46]: My bad—I have a fix out at https://github.com/linkedin/datahub/pull/4253

**Pull request merged**